packages/neutrino/src/neutrino.js

```diff
@@ -23,6 +24,34 @@ class Neutrino extends EventEmitter {
     this.options = merge(options, { root, source, output, tests, node_modules, entry });
   }
 
+  requirePresets(presets) {
+    return presets.map(preset => this.requirePreset(preset));
+  }
+
+  requirePreset(preset) {
+    const paths = [ join(cwd, preset), join(cwd, 'node_modules', preset), preset ];
```

Comment:
If we use `options.root`, which is usually `process.cwd()`, we shouldn't need `cwd`. Remove inner spaces from the array brackets. Paths is intermediary, so we can remove the variable:

```js
const module = [
  join(this.options.root, preset),
  join(this.options.root, 'node_modules', preset),
  preset
].find(path => this.requirePath(path));
```